enterprise/backend/src/metabase_enterprise/audit_app/pages/downloads.clj

```diff
@@ -55,14 +55,14 @@
 
 (def ^:private bucket-maxes
   "Add/remove numbers here to adjust buckets returned by the `by-size` query."
-  [     10
-       100
-      1000
-      5000
-     10000
-     50000
-    100000
-    500000
+  [10
+   100
+   1000
+   5000
+   10000
+   50000
+   100000
+   500000
```

Comment:
This sort of thing really makes me wish https://github.com/weavejester/cljfmt/issues/78 was implemented